gp-includes/formats/format_pomo.php

```diff
@@ -45,6 +51,36 @@ public function read_originals_from_file( $file_name ) {
 		return $this->read_translations_from_file( $file_name );
 	}
 
+	/**
+	 * Create a string that represents the value for the "Language:" header for a po file.
+	 *
+	 * @since 2.1.0
+	 *
+	 * @param GP_Locale $locale The locale object.
+	 *
+	 * @return string|false
+	 */
+	private function generate_language_string( $locale ) {
+		$ret = '';
+
+		if ( $locale->lang_code_iso_639_1 ) {
+			$ret = $locale->lang_code_iso_639_1;
+		} else if ( $locale->lang_code_iso_639_2 ) {
+			$ret = $locale->lang_code_iso_639_2;
+		} else if ( $locale->lang_code_iso_639_3 ) {
+			$ret = $locale->lang_code_iso_639_3;
+		}
+
+		if ( '' === $ret ) {
+			return false;
+		}
+
+		if ( null !== $locale->country_code && strcasecmp( $ret, $locale->country_code ) === 0 ) {
+			$ret .= '_' . $locale->country_code;
```

Comment:
In `GP_Locales` the country code is lowercase, but the specification says "‘ll_CC’, where ‘ll’ is an ISO 639 two-letter language code (lowercase) and **‘CC’ is an ISO 3166 two-letter country code (uppercase).**":
`$ret .= '_' . strtoupper( $locale->country_code )`